app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3 or command[0] != "mv":
+        raise ValueError(f"Invalid command: {command}")
```

Comment:
Ensure that the error message uses consistent double quotes as per the code style checklist.